packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-identifier.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { type WardPatientCardElement } from '../../types';
+import { type PatientCardElementConfig } from '../../config-schema';
+import { Tag } from '@carbon/react';
+
+const identifierCompareFunction = (patientIdentifier1, patientIdentifier2) => {
+  const sortByDate =
+    (!patientIdentifier1.preferred && !patientIdentifier2.preferred) ||
+    (patientIdentifier1.preferred && patientIdentifier2.preferred);
+  if (sortByDate) {
+    const patientIdentifier2Date = new Date(
+      patientIdentifier2.auditInfo.dateChanged || patientIdentifier2.auditInfo.dateCreated,
+    );
+    const patientIdentifier1Date = new Date(
+      patientIdentifier1.auditInfo.dateChanged || patientIdentifier1.auditInfo.dateCreated,
+    );
+    return patientIdentifier2Date.valueOf() - patientIdentifier1Date.valueOf();
+  }
+  if (patientIdentifier1.preferred) return -1;
+  if (patientIdentifier2.preferred) return 1;
+  return 0;
+};
+
+const wardPatientIdentifier = (config: PatientCardElementConfig) => {
+  const WardPatientIdentifier: WardPatientCardElement = ({ patient }) => {
+    const { identifierTypeUuid } = config;
+    const patientIdentifiers = patient.identifiers.filter(
+      (patientIdentifier) => patientIdentifier.identifierType?.uuid === identifierTypeUuid,
+    );
+    patientIdentifiers.sort(identifierCompareFunction);
+    const patientIdentifier = patientIdentifiers[0];
+
+    return (
+      <div>
+        <Tag>{`${patientIdentifier?.identifierType.name}:`}</Tag>
```

Comment:
I would not assume that we always want to display the identifier type name, and even if we did want to always have a label, identifierType name may not be appropriate as it is not localized.

We will want to do something like what @chibongho is doing with the obs element, where one can configure a custom label, or no label, or choose (via configuration) to use the type as the label.

Reply:
hi @mseaton , i have made the changes for displaying patient identifier with the  value specified in config. Please let me know for any other changes. Thank you.